### PR TITLE
Add documentation for bastion

### PR DESCRIPTION
**Pull request by MalloZup**
add minimal doc for documenting bastion feature

### CODE REVIEW

File: azure/README.md [hunk 1]
```diff
@@ -59,12 +59,20 @@ terraform plan
 terraform apply
 ```
 
-Connect using `ssh` as the user set as your `admin_user` parameter, for example:
+### Bastion
 
+By default it is enabled the bastion in azure (you can still disable it), which has the only public ip. Connect via `ssh` as cloudadmin user and your cloud ssh key:
```
Comment (arbulu89) [inline azure/README.md:64]: I would rephrase this to: `By default, the bastion machine is enabled in Azure (it can be disabled), which will have the unique public IP address of the deployed resource group. Connect using `ssh` and the selected admin user with:
```ssh {admin_user}@BASTION_IP -i {private_key_location}```
`

File: azure/README.md [hunk 2]
```diff
@@ -59,12 +59,20 @@ terraform plan
...
+```ssh cloudadminr@BASTION_IP -i private_key_location```
+
+If you want to log to hana and others instances, use:
```
Comment (arbulu89) [inline azure/README.md:67]: Maybe better, `To log to HANA and other instances, use`

File: azure/README.md [hunk 3]
```diff
@@ -59,12 +59,20 @@ terraform plan
...
 ```
-ssh admin_user@18.196.143.128 -i private_key_location
+ssh -o ProxyCommand="ssh -W %h:%p cloudadmin@40.113.121.182 -i /home/YOUR_USER/.ssh/id_rsa_cloud -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" cloudadmin@10.74.1.10 -i /home/YOUR_USER/.ssh/id_rsa_cloud -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
 ```
 
+If you want to disable the bastion use:
```
Comment (arbulu89) [inline azure/README.md:72]: `To disable the bastion:`

File: azure/README.md [hunk 4]
```diff
@@ -59,12 +59,20 @@ terraform plan
...
-Connect using `ssh` as the user set as your `admin_user` parameter, for example:
+### Bastion
 
+By default it is enabled the bastion in azure (you can still disable it), which has the only public ip. Connect via `ssh` as cloudadmin user and your cloud ssh key:
+```ssh cloudadminr@BASTION_IP -i private_key_location```
+
+If you want to log to hana and others instances, use:
 ```
-ssh admin_user@18.196.143.128 -i private_key_location
+ssh -o ProxyCommand="ssh -W %h:%p cloudadmin@40.113.121.182 -i /home/YOUR_USER/.ssh/id_rsa_cloud -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" cloudadmin@10.74.1.10 -i /home/YOUR_USER/.ssh/id_rsa_cloud -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
```
Comment (arbulu89) [inline azure/README.md:69]: Update the command and remove `cloudadmin`. This is a configurable value

**Review by arbulu89 — CHANGES_REQUESTED** [review]
@MalloZup Some comments

File: azure/README.md (hunk 2 above)
Comment (MalloZup) [reply azure/README.md:67]: ok

**Comment by MalloZup** [comment]
done thx

File: azure/README.md [hunk 5]
```diff
@@ -60,12 +60,19 @@ terraform plan
 terraform apply
 ```
 
-Connect using `ssh` as the user set as your `admin_user` parameter, for example:
+### Bastion
 
+By default, the bastion machine is enabled in Azure (it can be disabled), which will have the unique public IP address of the deployed resource group. Connect using ssh and the selected admin user with: ```ssh {admin_user}@BASTION_IP -i {private_key_location}
```
Comment (arbulu89) [inline azure/README.md:65]: 3 quotes missing at the end of the line to close the command

**Review by arbulu89 — CHANGES_REQUESTED** [review]

File: azure/README.md (hunk 5 above)
Comment (MalloZup) [reply azure/README.md:65]: doh

Comment (MalloZup) [reply azure/README.md]: done thx

File: azure/README.md [hunk 6]
```diff
@@ -60,12 +60,19 @@ terraform plan
...
 
-Connect using `ssh` as the user set as your `admin_user` parameter, for example:
+### Bastion
 
+By default, the bastion machine is enabled in Azure (it can be disabled), which will have the unique public IP address of the deployed resource group. Connect using ssh and the selected admin user with: ```ssh {admin_user}@BASTION_IP -i {private_key_location}```
+
+To log to hana and others instances, use:
 ```
-ssh admin_user@18.196.143.128 -i private_key_location
+ssh -o ProxyCommand="ssh -W %h:%p ADMIN_USER@BASTION_IP -i /home/YOUR_USER/.ssh/id_rsa_cloud -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" ADMIN_USER@BASTION_IP -i /home/YOUR_USER/.ssh/id_rsa_cloud -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
```
Comment (arbulu89) [inline azure/README.md:69]: I would have the same syntax in the 1st and 2nd commands. I mean:
- Same way of defining `admin_user` -> `{admin_user}`
- Same way of defining `bastion_ip` -> `{bastion_ip}`
- Same way of defining `private_key_location` -> `{private_key_location}

**Review by arbulu89 — APPROVED** [review]
@MalloZup I'm approving the PR to avoid more reviews, but I have commented about the commands formatting. Both commands should follow the same syntax. Have a look.

File: azure/README.md (hunk 6 above)
Comment (MalloZup) [reply azure/README.md:69]: ok will check tomorrow

Comment (MalloZup) [reply azure/README.md]: done thx
